src/spec/determineConfig.spec.js

```diff
@@ -168,11 +194,13 @@ describe("determineConfig", function() {
           snapshotFilename: "filename",
           snapshotNameTemplate: "name",
         },
-        envFlag: true,
+        envUpdateFlag: true,
+        envCiFlag: true,
```

Comment:
It's a bit pedantic, but to cover all our bases, we should also test when envUpdateFlag is false and envCiFlag is true, update should be false and ci should be true